tests/input/cpp/namespace_namespace.cpp

```diff
@@ -0,0 +1,6 @@
+namespace hw { namespace stm32 {
+
+class RTC {
+}
```

Comment:
missing semicolon here even if I don't think it'll change the formatting but it's good to test with valid C++.